src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/Fat/Search/FatSearchViewComponent.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Sfa.Das.Sas.Shared.Components.ViewModels;
 
 namespace Sfa.Das.Sas.Shared.Components.ViewComponents.Fat
 {
```

Comment:
Should the Invoke method by synchronous in here?

Reply:
its called using the async method in the view so not sure it makes much difference for us either way